packages/engine/src/executor.ts

```
@@ -3149,7 +3175,418 @@ export class TaskExecutor {
+  ): Promise<WorkflowNodeResult> {
+    const cfg = node.config ?? {};
+    const live = await this.store.getTask(nodeTask.id);
+
+    // Await-input nodes never run a session — they pause for the user.
+    if (cfg.awaitInput === true) {
+      return this.runAwaitInputNode(node, live);
+    }
+
+    const worktreePath = live.worktree || this.rootDir;
+    const executorKind = typeof cfg.executor === "string" ? cfg.executor : "model";
+    let scriptName = typeof cfg.scriptName === "string" && cfg.scriptName.trim() ? cfg.scriptName : undefined;
```

ESLint

[error] 'scriptName' is never reassigned. Use 'const' instead.

(prefer-const)